Clang warning explicitly defaulted constructor is deleted

Clang warns about explicitly defaulted constructors being implicitly deleted:
```
src/Things/Structures/SeedLander.h:12:2: warning: explicitly defaulted default constructor is implicitly deleted [-Wdefaulted-function-deleted]
        SeedLander() = default;
        ^
src/Things/Structures/SeedLander.h:6:19: note: default constructor of 'SeedLander' is implicitly deleted because base class 'Structure' has a deleted default constructor
class SeedLander: public Structure
                  ^
src/Things/Structures/../../Map/../Things/Structures/Structure.h:161:2: note: 'Structure' has been explicitly marked deleted here
        Structure() = delete;
        ^
```

The same warning is generated about 8 times, since it's in a header included by a number of other files.

--- a/src/Things/Structures/SeedLander.h
+++ b/src/Things/Structures/SeedLander.h
@@ -9,7 +9,7 @@ public:
 	typedef NAS2D::Signals::Signal2<int, int> Callback;
 
 public:
-	SeedLander() = default;
+	SeedLander() = delete;
 	SeedLander(int x, int y):	Structure(constants::SEED_LANDER, "structures/seed_0.sprite", CLASS_LANDER),
 								mX(x), mY(y)
 	{

